CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
   @cosmjs/launchpad instead.
 - @cosmjs/launchpad: Add `Secp256k1Wallet` to manage a single raw secp256k1
   keypair.
+- @cosmjs/launchpad: `OfflineSigner` type’s `sign` method renamed `signAmino`
+  and `SignResponse` type renamed `AminoSignResponse`.
+- @cosmjs/launchpad: `Secp256k1HdWallet.sign` method renamed `signAmino`.
```

Comment:
This needs to be done for `Secp256k1Wallet` as well. I assume it was done already, otherwise we get compile fails.

Reply:
Yes, I updated it but didn't add a changelog entry since it's also new in 0.24.

Reply:
Oh, right, perfect